app/data/mounts.json

```diff
@@ -291,6 +298,13 @@
             "spellid": 342334,
             "icon": "inv_wingedlion2mount",
             "itemId": 183740
+          },
+          {
+            "ID": 1382,
+            "name": "Inquisition Intimidator",
```

Comment:
This should go in the Venthyr covenant, as it's more restrictive than "reputation". All people in the Venthyr covenant can buy it if they get exalted, but not every exalted people can get it since they need to be in the correct covenant.

Reply:
Moved to Venthyr.